mllib/src/main/scala/org/apache/spark/ml/classification/OneVsRest.scala

```diff
@@ -271,7 +273,7 @@ object OneVsRestModel extends MLReadable[OneVsRestModel] {
 @Since("1.4.0")
 final class OneVsRest @Since("1.4.0") (
     @Since("1.4.0") override val uid: String)
-  extends Estimator[OneVsRestModel] with OneVsRestParams with MLWritable {
+  extends Estimator[OneVsRestModel] with OneVsRestParams with HasParallelism with MLWritable {
 
```

Comment:
It will be used by cross validator and train validation split, that's why it's shared.